projects/optic/src/config.ts

```diff
@@ -43,6 +43,7 @@ const RequestSend = Type.Object({
     })
   ),
   data: Type.Optional(Type.Object({})),
+  headers: Type.Optional(Type.Array(Type.Record(Type.String(), Type.String()))),
```

Comment:
Should this instead be just a record? I don't think we need to send multiple arrays of headers

e.g. `{ header-1: asd, header-2: 123 }` if you want to specify multiple